src/tests/class.test.js

```diff
@@ -1,5 +1,121 @@
-// Basic test to avoid conflicts
-// REMOVE THIS TEST AND MAKE YOURS!
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+import request from 'supertest';
+import app from '../app';
+import Class from '../models/Class';
+import classSeed from '../seeds/class';
+
+beforeAll(async () => {
+  await Class.collection.insertMany(classSeed);
+});
+afterEach(() => { jest.restoreAllMocks(); });
+
+describe('getClassById /api/class/:id', () => {
+  test('should return status 200 when id exists', async () => {
+    const id = '646004aff33f9c83d28ed954';
+    const response = await request(app).get(`/api/class/${id}`).send();
+    expect(response.status).toBe(200);
+    expect(response.body.error).toBe(false);
```

Comment:
We can change this line in order to keep the consistency.

```suggestion
    expect(response.body.error).toBeFalsy;
```